### DISCUSSION TITLE
Upgrading Axmol versions in project

### DISCUSSION THREAD

**Discussion by chadwyk** [axmolengine/axmol#1299] [General]
Is there a documented process or script available to update Axmol in my Xcode project to the latest version once I have it installed?  Or is it just best to just create a new project and import all my classes and media content every time a new version is released?

**Comment by rh101** [comment]
For minor versions updates (for example, updating from 1.0 to 1.1), just update your cloned axmol to the latest version and checkout the relevant branch/tag.  It would also be best to check if the project template files (such as [these](https://github.com/axmolengine/axmol/tree/1.x/templates/cpp-template-default)) have been modified since the last version you used, and merge the changes in to your own project if required.
